openmetadata-ui/src/main/resources/ui/src/components/Glossary/GlossaryHeader/GlossaryHeader.component.tsx

```diff
@@ -554,12 +554,8 @@ const GlossaryHeader = ({
 
   return (
     <>
-      <Row
-        className="glossary-header"
-        gutter={[0, 16]}
-        justify="space-between"
-        wrap={false}>
-        <Col className="d-flex" flex="auto">
+      <div className="glossary-header flex gap-2 justify-between no-wrap ">
```

Comment:
```suggestion
      <div className="glossary-header flex gap-4 justify-between no-wrap ">
```
To preserve older spacing, it should be gap-4, no?